README.md

```diff
@@ -33,9 +33,9 @@ The job of the connector is to read messages from the topic, invoke a HTTP endpo
 
 ```
 {
-   "X-Fission-MQTrigger-Topic": Topic,
-   "X-Fission-MQTrigger-RespTopic": ResponseTopic,
-   "X-Fission-MQTrigger-ErrorTopic": ErrorTopic,
+   "Topic": Topic,
+   "RespTopic": ResponseTopic,
+   "ErrorTopic": ErrorTopic,
```

Comment:
Although the X header names are deprecated, these headers can easily collide with something else. I suggest some format like `x-keda-topic` or just `keda-source-topic`

Reply:
Of course, we will have to change in the actual connector code I think?

Reply:
Yes makes sense